examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/README.md

```diff
@@ -1,179 +1,96 @@
-Step-by-Step
-============
-This document describes the step-by-step instructions to run large language models (LLMs) on 4th Gen Intel® Xeon® Scalable Processor (codenamed Sapphire Rapids) with PyTorch and Intel® Extension for PyTorch.
+Weight-only quantization
+===============
 
-The script `run_clm_no_trainer.py` supports `GPTJ`, `OPT`, `LLaMA2`, `BLOOM` and `Falcon` quantization and validates last word prediction accuracy with [lm_eval](https://github.com/EleutherAI/lm-evaluation-harness.git) now, and we are adding more models.
-
-# Prerequisite
-## 1. Create Environment
+##  Prerequisite
 ```
 # Installation
 pip install -r requirements.txt
 ```
 
-# Run
+## Support status on HPU
 
-Here is how to run the scripts:
+Below is the current support status on Intel Gaudi 2 AI Accelerator with PyTorch.
```

Comment:
no need to call out Gaudi 2,  can use Intel Gaudi AI Accelerator

Reply:
I think it is ok to generally mention Gaudi AI Accelerator, similar to our main page. we have release notes call out validated HW Gaudi 2 and 3